lib/graphql.module.ts

```diff
@@ -46,6 +47,7 @@ import {
     GraphQLTypesLoader,
     GraphQLSchemaBuilder,
     GraphQLSchemaHost,
+    { provide: APP_FILTER, useClass: GraphQLExceptionFilter },
```

Comment:
This will bind the filter for the entire application. If someone is using GQL and REST they are now using the `GraphQLExceptionFilter` for both contexts

Reply:
I've added a check in `GraphQLExceptionFilter` to rethrow exceptions that were thrown not in GraphQL context.
If you think it make sense, I can go ahead and add a test for that specifically. I'd probably need to add a REST controller to the *code-first* test app, does it make sense?

Reply:
To avoid introducing breaking changes, let's make this optional for now and inform users that they have to explicitly register this filter in their applications. Later (in v8), we can get back to this discussion and think about registering it automatically.

Reply:
Sure. I've added it manually to *guards-filters.spec.ts* and *pipes.spec.ts* as these are the tests that throw exceptions.